espaloma/nn/readout/charge_equilibrium.py

```diff
@@ -48,7 +48,7 @@ def get_charges(node):
     sum_q = node.data["sum_q"]
 
     return {
-        "q_hat": -e * s ** -1
+        "q": -e * s ** -1
```

Comment:
Can we use more informative variable names, like "charge"? Also, can we explicitly document what the units are supposed to represent for the object model in some central location?